src/Microsoft.DotNet.Cli.Utils/FixedPathCommandFactory.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class FixedPathCommandFactory : ICommandFactory
```

Comment:
Yes, but the way we have our Command.Create methods setup, there is no way for the caller to know that. It needs to pass everything and then pass along to the resolvers that care.